ext/curl/interface.c

```diff
@@ -453,7 +453,7 @@ static zend_object *curl_clone_obj(zend_object *object) {
 	clone_ch->cp = cp;
 	_php_setup_easy_copy_handlers(clone_ch, ch);
 
-	postfields = &clone_ch->postfields;
+	postfields = &ch->postfields;
```

Comment:
Isn't the actual problem that we don't clone the `->postfields` in the first place? I.e. `ZVAL_COPY(&clone_ch->postfields, &ch->postfields)`.

Reply:
This code should behave consistently with PHP_FUNCTION(curl_copy_handle), with the only difference being the return value. This PR primarily aims to fix this bug, and a subsequent PR for code optimization can be submitted to implement a reusable function.

https://github.com/php/php-src/blob/5d10a33479c5a9bd71084f25a0b8e244ef7299da/ext/curl/interface.c#L1598

Reply:
If you clone an object, you should clone all its properties.

Reply:
I have modified the code to use ZVAL_COPY for postfields and updated PHP_FUNCTION(curl_copy_handle) to ensure consistency with the clone operation.